src/components/Wizard/CreateVmWizard/providers/v2vvmwareController.js

```diff
@@ -0,0 +1,169 @@
+import { DeploymentModel, RoleModel, ServiceAccountModel, RoleBindingModel } from '../../../../models';
+import { V2VVMWARE_DEPLOYMENT_NAME } from '../constants';
+
+// eslint-disable-next-line no-console
+const rejectLogger = title => reason => console.warn(`Failed to create ${title}, reason`, reason);
+
+// The controller is namespace-scoped, especially due to security reasons
+// Let's make sure its started within the desired namespace (which is not by default).
+// The V2VVmware CRD is expected to be already created within the cluster (by Web UI installation)
+export const startV2VVMWareController = async ({ k8sCreate, k8sGet, namespace }) => {
+  try {
+    await k8sGet(DeploymentModel, V2VVMWARE_DEPLOYMENT_NAME, namespace);
+  } catch {
+    // Deployment does not exist
```

Comment:
it may also mean that user does not have permissions to get Deployments

Reply:
User will need to be informed i.e. using toast notification, recent logging to console is not enough.
Due to complexity of the flow, let's move that to a follow-up.

Reply:
Follow-up issue: #240